daliuge-translator/dlg/dropmake/cwl.py

```diff
@@ -23,6 +23,7 @@
 import logging
 import os
 from zipfile import ZipFile
+import io
```

Comment:
Can this be inserted in the correct alpha-sorted position?

Reply:
No longer required, but I'll make sure to do this in future.